enhancements/installer/packet-ipi.md

```diff
@@ -0,0 +1,294 @@
+  images](https://docs.openshift.com/container-platform/4.5/installing/installing_bare_metal/installing-bare-metal.html#registry-configuring-storage-baremetal_installing-bare-metal)
+  ([mirror](https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.5/?extIdCarryOver=true&sc_cid=701f2000001OH7iAAG)).
+  For this to be viable, Packet would need to provide a mirror cache.
+
+  A generic RHCOS Ignition URL could instruct nodes booted on Packet to
+  configure their network using the EC2-like [metadata
+  service](https://www.packet.com/developers/docs/servers/key-features/metadata/).
+
+- Packet does not offer a managed DNS solution. Route53, CloudFlare, or RFC 2136
+  (DNS Update), and NS record based zone forwarding are considerations. Some
+  existing IPI drivers ([such as
+  `baremetal`](https://github.com/openshift/installer/blob/master/docs/user/metal/install_ipi.md))
+  have similar limitations.
+- Packet does not offer a loadbalancer service, but does [offer the networking
+  features](https://www.packet.com/developers/docs/network/) that makes this
+  possible at the device level.
+- [Packet CCM currently offers
+  LoadBalancing](https://github.com/packethost/packet-ccm#load-balancers) by
+  creating and maintaining a MetalLB deployment within the cluster.  This may be
+  impacted by <https://github.com/openshift/enhancements/pull/356>.
+- [Packet
+  CSI](https://github.com/packethost/csi-packet#kubernetes-container-storage-interface-csi-plugin-for-packet)
+  currently requires a minimum of 100GB per volume. Additionally, the [Packet
+  EBS service](https://www.packet.com/developers/docs/storage/ebs/) is available
+  in a limiting subset of the available facilities.
+- Both Public and Private IPv4 and IPv6 addresses made available for each node.
+  Private and VLAN Networks can be restricted at the project level (default) or
+  node by node. While custom ASNs and BGP routing features are supported, these
+  and other networking features will not be exposed through the IPI.
+  <https://www.packet.com/cloud/network/>
+- Openshift will run on RHCOS directly on bare metal without the need for a
+  VM layer.
+
+### Risks and Mitigations
+
+The [official Packet
+ClusterAPI](https://github.com/kubernetes-sigs/cluster-api-provider-packet),
+[CCM](https://github.com/packethost/packet-ccm), and
+[CSI](https://github.com/packethost/csi-packet) drivers are all relatively new
+and may make undertake substantial design changes.  This project will either
+need to adopt and maintain the current releases, or adopt newer releases
+developed alongside this project.
+
+The CI will need to assure that Packet resources are cleanly released after
+failed and successful builds.  Packet is currently used in the bare metal
+provisioning tests, so there is some prior art available.  If the same account
+is used for testing the IPI, it may need account quota increases.
+
+## Design Details
+
+### Open Questions
+
+- Is the Packet CCM necessary?
```

Comment:
Does it provide features beyond deploying and configuring MetalLB?

Reply:
I've added a better description here.